storybook/pages/CommunityPermissionsSettingItemEditor.qml

```diff
@@ -0,0 +1,105 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Storybook 1.0
+import Models 1.0
+
+ColumnLayout {
+    id: root
+
+    property string panelText
+    property string name
+    property string icon
+    property string amountText
+    property bool isAmountVisible: false
+    property bool isExpression: false
+    property bool isAnd: true
+    property bool isImageSelectorVisible: true
+    property var iconsModel
+
+    Label {
+        Layout.fillWidth: true
+        text: root.panelText
+        font.weight: Font.Bold
+    }
+
+    RowLayout {
+        Rectangle {
+            id: image
+            visible: root.isImageSelectorVisible
+            border.color: 'lightgrey'
+            radius: 16
+            Layout.preferredHeight: 50
+            Layout.preferredWidth: 50
+
+            Image {
+                anchors.fill: parent
+                anchors.margins: 1
+                fillMode: Image.PreserveAspectFit
+                source: root.icon
+            }
+
+            MouseArea {
+                anchors.fill: parent
+                onClicked: iconSelector.open()
+
+                ImageSelectPopup {
+                    id: iconSelector
+
+                    parent: root
+                    anchors.centerIn: parent
+                    width: 200
+                    height: 250
+
+                    model: root.iconsModel
+
+                    onSelected: {
+                        root.icon = icon
+                        close()
+                    }
+                }
+            }
+        }
+
+        ColumnLayout {
+            Label {
+                Layout.fillWidth: true
+                text: "Type"
+            }
+            TextField {
+                background: Rectangle {
```

Comment:
In that case maybe we can agree in a basic basic basic style to apply in some editors, otherwise it will be really difficult to use. For example, in this case, that we have a hierarcy, for me it was difficult to leave it without style bc if not it became difficult for the user to understand which is the item he/she is trying to edit.. WDYT?